onmt/inputters/corpus.py

```diff
@@ -229,7 +232,9 @@ def _add_index(self, stream):
 
     def _iter_corpus(self):
         corpus_stream = self.corpus.load(
-            stride=self.stride, offset=self.offset)
+            stride=self.stride, offset=self.offset,
+            log_level=self.skip_empty_level
```

Comment:
not very straightforward to use `skip_empty_level` here...
maybe rename this flag to `data_loading_level` (and retain backcompatibility by keeping `skip_empty_level` but assigning its value to `data_loading_level` + warn for deprecation)